index.js

```diff
@@ -141,3 +142,12 @@ module.exports = (archive, target, opts, cb) => {
   return status
 }
 
+function joinHyperPath (base, path) {
+  path = join(base, path)
+  if (path === '.') {
+    // '.' is returned when base is '' and path is '': aka, the root directory
+    // in hyperdrive, root should be '' or '/', so we replace it with this special case
+    return ''
+  }
+  return path
```

Comment:
Which wouldn't be a big deal, for me anyway. The ecosystem is inconsistent about the leading slash. All of my consuming code now normalizes the value.